### PR TITLE
Code improvement

### PR DESCRIPTION
**Pull request by leonardosnt**

### CODE REVIEW

File: Class1.cs [hunk 1]
```diff
@@ -48,57 +32,45 @@ private void UnturnedPlayerEvents_OnPlayerDeath(Rocket.Unturned.Player.UnturnedP
...
+                    case EDeathCause.ROADKILL:
+                        UnturnedChat.Say(Rocket.Unturned.Player.UnturnedPlayer.FromCSteamID(murderer).CharacterName + " " + this.Configuration.Instance.roadkill + " " + player.CharacterName + "!", Color.red);
+                        break;
+                    case EDeathCause.VEHICLE:
+                        UnturnedChat.Say(player.CharacterName + " " + this.Configuration.Instance.vehicle, Color.red);
+                        break;
                 }
             }
         }
-        protected override void Unload()
```
Comment (leonardosnt) [inline Class1.cs]: Redundant method override.

Comment (leonardosnt) [inline Class1.cs]: Redundant method override

File: Class1.cs [hunk 2]
```diff
@@ -1,22 +1,14 @@
...
 using Rocket.Core.Plugins;
 using Rocket.Core.Logging;
 using Rocket.Unturned.Chat;
+using SDG.Unturned;
 
 //Yeha, we are on GitHub
 namespace Sadusko.DeathMessages
 {
-    public class PlayerDeath : RocketPlugin<DMC>
+    public class PlayerDeath : RocketPlugin<DMCconf>
```
Comment (leonardosnt) [inline Class1.cs]: Fix

Comment (Sadusko) [inline Class1.cs:11]: Actually why would I, this is not "improvements", this is basically changing.

DMC means DeathMessagesConfiguration.

**Comment by leonardosnt** [comment]
https://github.com/Sadusko/Death-Messages/pull/1/files#diff-6eb1df990ad477b50929de9c8a3c564bR37

**Comment by Sadusko** [comment]
leonardosnt. I have this incase of someone who is wanting to fix something that isn't fixed.
Everything you have added is basically executing the same performance.
So please.. lol

**Pull request closed without merging**